src/Core/Commander/ManagerCommands.js

```diff
@@ -40,6 +40,9 @@ define('Core/Commander/ManagerCommands', [
             this.providerMap = {};
             this.history = null;
             this.eventsManager = new EventsManager();
+            this.currentCommandsCount = 0;
+            this.maxConcurrentCommands = 16;
```

Comment:
`setImmediate` is non-standard so maybe we should prefer a standard based implementation.
This post https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/Hn3GxRLXmR0/XP9xcY_gBPQJ explains why Chrome didn't implement `setImmediate` but also details several alternatives.
The last `tick` function in his post matches what we need; it's just missing a call to `requestAnimationFrame`